mime_types.h

```diff
@@ -0,0 +1,23 @@
+//
+// mime_types.hpp
+// ~~~~~~~~~~~~~~
+//
+// Copyright (c) 2003-2012 Christopher M. Kohlhoff (chris at kohlhoff dot com)
+//
+// Distributed under the Boost Software License, Version 1.0. (See accompanying
+// file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+//
+
+#ifndef HTTP_MIME_TYPES_HPP
+#define HTTP_MIME_TYPES_HPP
+
+#include <string>
+
+namespace mime_types {
```

Comment:
Can we re-format this to eliminate the newlines and indent over the items in the namespace? That is: 

```c++
namespace mime_types {
  // Convert a file extension into a MIME type.
  std::string extension_to_type(const std::string& extension);
} // namespace mime_types
```

There's also a triple comment `///`?